### DISCUSSION TITLE
What's Next?

### DISCUSSION THREAD

**Discussion by alunduil** [alunduil/etest#69] [General]
It looks like we've solved the outstanding issues for etest and the project structure is a bit more understandable.  I still see the following potential changes that would make the project a bit easier to work with or follow:

- [x] Switch to git flow so we can easily manage release metadata in commits and have a flow for controlling prelease found issues.
- [x] Simplify the test suite immensely.
- [x] Publish this package to PyPi automatically so we can have autogenerated packages in many distributions.
- [ ] Ensure that this is fit for purpose by running it against ebuilds.
- [ ] Improve the test selection logic to make it better at determining what USE flags might be best to test against.
- [ ] Collect usage data and test outcomes in a centralised location to improve the tests for every run (overridable of course).

These are just some ideas and hopefully we can figure out what things might be useful but as long as it runs and someone finds it useful it's good enough I say.

**Comment by alunduil** [comment]
@LinoBigatti let me know what you think of this and we should probably discuss maintainership moves at some point as well.

**Comment by LinoBigatti** [comment]
I would probably also add more verbose logging, because currently it seems like ebuilds silently fail, and it could be a myriad of things. We need to debug that asap. Also an ebuild on ::guru could be good once we get it up and running.

**Reply by alunduil** [reply]
Those are great additions.

**Comment by LinoBigatti** [comment]
We could also add a project entry to keep track of progress.

**Reply by alunduil** [reply]
I was just thinking this. If you haven't created one by the time I get a chance to poke at things I'll go ahead and do it.

**Reply by LinoBigatti** [reply]
Go for it, Im away from home
